lib/actions/tags.ts

```diff
@@ -267,10 +290,13 @@ export const remove: CommandDefinition<
 				return balena.models.device.tags.remove(options.device, params.tagKey);
 			}
 			if (options.release) {
-				return balena.models.release.tags.remove(
+				const releaseParam = await disambiguateReleaseParam(
+					balena,
 					options.release,
-					params.tagKey,
+					options.release_raw!,
```

Comment:
Instead of the exclamation mark here, it may be possible to use a question mark (`paramRaw?`) in the declaration of `disambiguateReleaseParam` -
```js
export async function disambiguateReleaseParam(
	balena: BalenaSDK,
	param: string | number,
	paramRaw?: string,
) {
```
Or not - it was just a thought.

Reply:
I was thinking about this as well, but I think we should better go for `paramRaw: string | undefined` instead of `?` since the arg isn't meant to be optional.